slack-go.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+define("GOSERVER", "http://go.brandwatch.com/");
+define("GOSERVERSHORT", "http://go/");
+define("GOSERVERDISPLAY", "go");
+define("BOTNAME", "go");
+define("INCOMINGWEBHOOK", "https://hooks.slack.com/services/xxx"); //Use your own incoming webhook URL here
+
+function get_long_url( $slug ) 
+{ 
+    $options = array( 
+        CURLOPT_RETURNTRANSFER => true,     // return web page 
+        CURLOPT_HEADER         => false,    // return headers 
+    ); 
+ 
+    $ch = curl_init( GOSERVER . '_api/go-link.json' ); 
```

Comment:
I couldn't decipher what `$ch` stands for. Would it be possible to have a more descriptive name for this variable?